x-pack/plugins/enterprise_search/public/applications/shared/doc_links/doc_links.test.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { docLinksService } from './doc_links';
+
+describe('DocLinksService#setDocLinks()', () => {
+  it('adds links from docLinks', () => {
```

Comment:
naming nit - if we ever add further methods to this helper, the test block should be more easily extensible

```suggestion
describe('DocLinksService', () => {
  it('setDocLinks', () => {
```